src/Data/HintableModelTrait.php

```diff
@@ -81,6 +85,10 @@ protected function getHintableProps(): array
 
             // IMPORTANT: check if all hintable property are not set, otherwise the magic functions will not work!
             foreach ($cls as $cl) {
+                if (interface_exists($cl)) {
```

Comment:
there should be no check for class/iface, even if such check produces the correct output

Reply:
without that it gives error because is an interface : "Cannot bind closure to scope of internal class Traversable"

Reply:
yes, you make it working 👍 , but ifaces should be put into another array and merged when they are needed together :)

Reply:
and no loop even needed: https://3v4l.org/m19WH